packages/frontend/src/features/sqlRunner/components/CartesianChartFieldConfiguration.tsx

```diff
@@ -221,18 +213,10 @@ const GroupByFieldAxisConfig = ({
                     );
                 }
             }}
-            icon={
-                field?.reference ? (
-                    <TableFieldIcon
-                        fieldType={
-                            sqlColumns?.find(
-                                (x) => x.reference === field?.reference,
-                            )?.type ?? DimensionType.STRING
-                        }
-                    />
-                ) : null
+            fieldType={
+                sqlColumns?.find((x) => x.reference === field?.reference)
```

Comment:
Why doesn't the option object have this information? Perhaps work for a separate PR.

Reply:
Absolutely. Adding a ticket: https://github.com/lightdash/lightdash/issues/11012